Source/WebCore/editing/Editor.h

```diff
@@ -128,6 +128,8 @@ enum class TemporarySelectionOption : uint8_t {
     RevealSelectionBounds = 1 << 6,
 
     UserTriggered = 1 << 7,
+    
+    ForceCenterScroll = 1 << 8,
```

Comment:
Fine, but would be nice if this used the wording from ScrollAlignment. *Ideally* we'd just pass a ScrollAlignment in, but for some reason we've (previously) gotten really hung up on just having flags here.